beetsplug/missing.py

```diff
@@ -181,8 +229,8 @@ def _missing_albums(self, lib, query):
                 continue
 
             try:
-                resp = musicbrainzngs.browse_release_groups(artist=artist[1])
-                release_groups = resp['release-group-list']
+                resp = musicbrainzngs.browse_releases(artist=artist[1])
+                release_groups = resp['release-list']
```

Comment:
I'm wondering whether the changes from this PR should be hidden behind a command line flag (`--show-releases`, `--detailed-releases`?). Going from release groups to individual releases will substantially increase the amount of output (I'd imagine you'd easily see 5-10 times as many output lines for many artists), making it harder to parse visually. I'm not a user of this plugin myself, but I think I'd mostly be interested in knowing the release groups that I'm missing rather than each and every Vinyl, CD and digital release.

What do you think?

Style-wise, I think `release_groups` (and `rg`) below should be renamed to `releases` and `r` to avoid confusion.

Reply:
Sorry for the confusion with the first commit, I should have tested more before opening the PR.
This change does increase the output, but the original `rg` object contains only release group id, artist, and album name.
If we want allow a formatting string that uses any other placeholder it's necessary to get the full album object.